lib/aca_entities/eligibilities/contracts/v3/eligibility_contract.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Eligibilities
+    module Contracts
+      module V3
+        # Schema and validation rules for {AcaEntities::Eligibilities::V3::Eligibility}
+        class EligibilityContract < Dry::Validation::Contract
+          params do
+            optional(:determinations).array(
+              AcaEntities::Eligibilities::Contracts::V3::DeterminationContract.params
+            )
+            optional(:evidences).array(
+              AcaEntities::Eligibilities::Contracts::V3::EvidenceContract.params
+            )
+            optional(:state_histories).array(
+              AcaEntities::Eligibilities::Contracts::V3::StateHistoryContract.params
+            )
+            optional(:key).maybe(:symbol)
+            optional(:title).maybe(Types::Coercible::String)
+            optional(:description).maybe(Types::Coercible::String)
+            optional(:current_state).maybe(:symbol)
+            optional(:is_satisfied).maybe(:bool)
+            optional(:determined_at).maybe(:date_time)
```

Comment:
we have is_satisfied + determined_at but we also have is_disqualified and disqualified_at but not is_satisfied_at. need to verify why we need the is_disqualified_at vs just using the determined_at